test/lib/CMakeLists.txt

```diff
@@ -1,19 +1,3 @@
-#
-# Copyright 2020 Autodesk
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
```

Comment:
Not clear on which CMakeLists.txt files should have a copyright header, and which shouldn't.  Can you shed some light on this?

Reply:
@elrond79 
We only need to add the `Copyright` only at the top level CmakeList.txt file for each mini projects:
e.g
```
maya-usd/CMakeLists.txt
maya-usd\lib\CMakeLists.txt
maya-usd\lib\usd\ui\CMakeLists.txt
maya-usd\lib\usd\ui\demo\CMakeLists.txt
```
Yes,  I will remove the copyright from `test/CMakeLists.txt` later. It is not really needed here.

Reply:
That, that clarifies things!